src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -70,6 +70,13 @@
   public static final Pattern GOOG_MODULE_EXTRACT =
       Pattern.compile("goog.module\\(['\"](.*)['\"]\\);");
 
+  /**
+   * Contains symbols that are part of platform externs, but not yet in lib.d.ts
+   * This list is incomplete and will grow as needed.
+   */
+  private static final Set<String> emittedPlatformSymbols = Sets.newHashSet(
+      "Image");
```

Comment:
added 8 other ones, which lead to 12 total, general_with_platform.d.ts bubbled a bit.